py/htm/examples/networkAPI/hotgym.py

```diff
@@ -0,0 +1,175 @@
+# ------------------------------------------------------------------------------
```

Comment:
Very nice example.

I would have used the Network.configure( ) function instead of using the 'default_parameters' and the addRegion( ) and Link( ) calls. It might be simpler but this works too.